### PR TITLE
Aztec integration

### PR DESCRIPTION
**Pull request by 0nko**
Introduces the Aztec editor to WordPress. Merged version is [0.5.0-alpha.2](https://github.com/wordpress-mobile/WordPress-Aztec-Android/releases/tag/v0.5.0-alpha.2).

Aztec is enabled by default for the alpha build but needs to be unlocked for beta by visiting the following URL: `wordpress://aztec?available=1&enabled=1`.

### CODE REVIEW

File: WordPress/build.gradle [hunk 1]
```diff
@@ -12,9 +12,11 @@ buildscript {
 
 repositories {
     jcenter()
+    mavenLocal()
```
Comment (maxme) [inline WordPress/build.gradle]: I'm not a big fan of having this in develop.

File: WordPress/src/main/res/values/key_strings.xml [hunk 2]
```diff
@@ -165,6 +165,18 @@
         <item>2000</item>
     </string-array>
 
+    <!-- App Settings - Editor -->
+    <string-array name="editor_values">
```
Comment (maxme) [inline WordPress/src/main/res/values/key_strings.xml]: We're missing `translatable="false"` here and in the other array below (sorry I didn't spot that one in previous PR).

File: libs/editor/WordPressEditor/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -94,4 +94,18 @@
     <string name="editor_dropped_title_images_not_allowed">Dropping images in the Title is not allowed</string>
     <string name="editor_dropped_html_images_not_allowed">Dropping images while in HTML mode is not allowed</string>
     <string name="editor_dropped_unsupported_files">Warning: not all dropped items are supported!</string>
+
+    <!-- AZTEC -->
+    <string name="edit_hint">Share your story here&#8230;</string>
+    <string name="source_hint">&lt;p&gt;Share your story here&#8230;&lt;/p&gt;</string>
```
Comment (maxme) [inline libs/editor/WordPressEditor/src/main/res/values/strings.xml]: Not blocking for merging this but I think we should add the `<p>` and `</p>` programmatically (and reuse `edit_hint`), I fear translators will make mistakes on this string.

**Comment by 0nko** [comment]
>Found an issue with the link dialog

I tested it on API 25 emulator and it doesn't do that. Seems like it's the older Android version only (happens on API 18).. I'll create an issue.

File: WordPress/src/main/res/values/key_strings.xml (hunk 2 above)
Comment (0nko) [inline WordPress/src/main/res/values/key_strings.xml]: I saw that but I thought it didn't matter since they're just numbers.

**Comment by maxme** [comment]
Something I missed from the previous review: 
On a Vanilla build:
* Clear app data (or uninstall / reinstall the app).
* Sign in.
* Go to App Settings, see "Legacy" is selected.
* Create new post, see the Visual editor is enabled.

Apparently we don't read the `AppPrefs.isVisualEditorEnabled()` flag when we load the App settings screen.

**Comment by 0nko** [comment]
Hey @maxme, it was caused by a logic that takes care of when you have Aztec set as an editor in alpha and then reinstall it with vanilla that doesn't have Aztec as an option. It's fixed now.

**Comment by maxme** [comment]
:shipit:

**Pull request merged**